src/System.Private.ServiceModel/tests/Scenarios/Client/ChannelLayer/DuplexChannelShapeTests.4.0.0.cs

```diff
@@ -12,7 +12,7 @@
 using System.Xml;
 using Xunit;
 
-public static class DuplexChannelShapeTests
+public static partial class DuplexChannelShapeTests
```

Comment:
Need to add " : ConditionalWcfTest" to this line to make it derive from the base class containing the ConditionalFact conditions.